Sources/tasks/MsgReport-Notify.php

```diff
@@ -174,7 +174,7 @@ public function execute()
 					'REPORTERNAME' => $this->_details['sender_name'],
 					'TOPICLINK' => $scripturl . '?topic=' . $this->_details['topic_id'] . '.msg' . $this->_details['msg_id'] . '#msg' . $this->_details['msg_id'],
 					'REPORTLINK' => $scripturl . '?action=moderate;area=reportedposts;sa=details;rid=' . $this->_details['report_id'],
-					'COMMENT' => $comment,
```

Comment:
Since this is the only place that `$comment` is used, we may as well remove 'lr.body` and `$comment` from the query above.